Makefile

```diff
@@ -8,40 +6,38 @@ else
 endif
 
 TARGETS=awesome
-BUILDLN=build
+BUILDDIR=build
 
-all: $(TARGETS) $(BUILDLN) ;
+all: $(TARGETS) ;
```

Comment:
My last comment is wrong: `cmake-build` is phony and the dependency chain is `all -> awesome -> cmake-build`, so `all` will always cause `make` to run in `build`.